### PR TITLE
doc: complete and update the subsystem logging level info table

### PR DESCRIPTION
**Pull request by scienceluo**
doc: complete and update the subsystem loging level info table

Signed-off-by: Luo Kexue <luo.kexue@zte.com.cn>

### CODE REVIEW

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 1]
```diff
@@ -200,59 +200,105 @@ to their default level or to a level suitable for normal operations.
...
++--------------------+-----------+--------------+
+| ``kstore``         |     1     |      5       |
++--------------------+-----------+--------------+
+| ``rocksdb``        |     4     |      5       |
++--------------------+-----------+--------------+
+| ``leveldb``        |     4     |      5       |
++--------------------+-----------+--------------+
+| ``memdb``          |     4     |      5       |
++--------------------+-----------+--------------+
+| ``kinetic``        |     1     |      5       |
```
Comment (tchaikov) [inline doc/rados/troubleshooting/log-and-debug.rst]: might want to remove the `kinetic` entry here. it's not built anymore.

File: doc/rados/troubleshooting/log-and-debug.rst [hunk 2]
```diff
@@ -200,59 +200,105 @@ to their default level or to a level suitable for normal operations.
...
 | ``javaclient``     |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``asok``           |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``throttle``       |     1     |      5       |
+| ``throttle``       |     1     |      1       |
++--------------------+-----------+--------------+
+| ``refs``           |     0     |      0       |
++--------------------+-----------+--------------+
+| ``xio``            |     1     |      5       |
```
Comment (tchaikov) [inline doc/rados/troubleshooting/log-and-debug.rst]: might want to remove the `kinetic` entry here. it's not built by default anymore.

File: doc/rados/troubleshooting/log-and-debug.rst (hunk 1 above)
Comment (scienceluo) [reply doc/rados/troubleshooting/log-and-debug.rst]: @tchaikov Thanks for your review.
BTW where can i find the default build rule about the loging entry?

Comment (tchaikov) [reply doc/rados/troubleshooting/log-and-debug.rst]: could you be more specific w.r.t. "build rule about the logging entry"?

Comment (scienceluo) [reply doc/rados/troubleshooting/log-and-debug.rst]: @tchaikov Ok, I update the table entry info from src/common/subsys.h 
So if we don't need "kinetic", maybe we should remove "SUBSYS(kinetic, 1, 5)" too ?

File: doc/rados/troubleshooting/log-and-debug.rst (hunk 2 above)
Comment (scienceluo) [reply doc/rados/troubleshooting/log-and-debug.rst]: @tchaikov Ok, updated.

**Comment by scienceluo** [comment]
@tchaikov After we merge this pr, some split line are missing. Did I miss something?   
From: http://docs.ceph.com/docs/master/rados/troubleshooting/log-and-debug/
default

**Comment by tchaikov** [comment]
no. it looks right.

**Pull request merged**